config.go

```diff
@@ -74,4 +75,14 @@ func init() {
 		// If $PORT is set, override LFS_LISTEN. This is useful for deploying to Heroku.
 		Config.Listen = "tcp://:" + port
 	}
+
+	if Config.ExtOrigin == "" {
+		// why not ignore the IsHTTPS check and use the following statement directly:
+		// `Config.ExtOrigin = fmt.Sprintf("%s://%s", Config.Scheme, Config.Host)`
```

Comment:
Using `Config.ExtOrigin = fmt.Sprintf("%s://%s", Config.Scheme, Config.Host)` seems fine to me if you want to make that change in another commit in this PR.  The only reason I can see not to do so would be if `Config.Scheme` could be empty in the default case, but the default is [set](https://github.com/git-lfs/lfs-test-server/blob/0ef89493762628a76d3f22a5e14fdf827ce80bc5/config.go#L22) as `http`, so that should be OK.

Reply:
That is true. The default value of `Config.Scheme` is `http`. So I prefer to use it directly to reduce further confustion. I will push the change to simplify the codes. Thank you so much for reply.